packages/uicore/src/components/Filters/components/FiltersMultiSelectDropDown/FiltersMultiSelectDropDown.css

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright 2024 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+.main {
+  &.disabled {
+    pointer-events: none;
+    .dropdownButton {
+      &.withBorder {
+        background-color: var(--grey-100);
+      }
+      .labelWrapper {
+        .label {
+          color: var(--grey-400) !important;
+        }
+      }
+    }
+    .counter {
+      background-color: var(--grey-400) !important;
+    }
+  }
+  .dropdownButton {
+    border-radius: var(--spacing-2);
+    padding: 0px var(--spacing-4);
+    height: var(--spacing-8);
+    transition: all 200ms ease-out;
+    transition-property: border-color, background-color;
+    &.minWidth {
+      column-gap: var(--spacing-small);
+    }
+    &.withBorder {
+      border: 1px solid var(--grey-200);
```

Comment:
The css was picked from existing component. Any improvements can be picked later